docs/introduction/exchanges.md

```diff
@@ -1,10 +1,50 @@
+
+### Transfer vs Process
+
+Transfers and Processes both structure "value flows", but they are conceptually and functionally different.
+* Processes create or change or transport an economic resource, whether producing goods or services, recycling, even if they create "waste" as a by-product (that can hopefully be used somewhere else).
+* Transfers do not create or change or transport an economic resource, they just put the rights and responsibilities for a resource into a different agent's hands so that the resource can be used or consumed or transfered again.
+
+This is an important distinction as we seek to change the world, because transfer and exchange cannot move us beyond the current unsustainable and unjust system.  We the people need to manage our production and creation (and waste) for human and ecological needs rather than profit.
+
+That said, the structure in value flows is similar, allowing transfers and processes to participate seamlessly in value flows:
+![transfer-process](https://rawgit.com/valueflows/valueflows/master/release-doc-in-process/transfer.png)
+
+### Process events as implied transfers
+
+Process related economic events can also imply a transfer, and can thus be used in an exchange.
+
+For example, most timebanks exchange work for credits.  The work event can be part of a process that produces something for some other agent.  It is also part of an exchange in the timebank.  The transfer of credits on the other hand, is not part of any process that creates or transports something, it is merely the recording of one account being decremented and one account being credited.
+
+Exchange of work also happens in open value networks, where people record work events as input to many processes, and then when income is received for outputs of that work, people receive part of that income, in exchange for their work.
+
+Another example is when a service is created as an output of a process, where that service delivery event can be considered an implied transfer, and exchanged for some other resource.
+
+### Agreements
+
+Various kinds of agreements between agents govern the rights and responsibilities for economic resources that are transferred. Agreements can reflect any economic paradigm, so make it possible for the Exchange and Transfer vocabulary to work equally well for capitalist businesses, transitional economic interactions, and next economy interactions.
+
+Agreements can be of any kind and scope, from an order to a value equation to a much larger blanket agreement.
+
+ValueFlows does not define the vocabulary for agreements, but defines optional references to an agreement when it could be useful.
```

Comment:
Actually we do define `vf:Agreement` and `vf:under`. So far we mostly used agreements to wrap commitments and events into a transaction. We don't have any example of using `vf:Agreement` for 'rights and responsibilities'.